generate_wordstr_box.py

```diff
@@ -44,6 +38,9 @@ def trim(im):
 # create WordStr line boxes for Indic & RTL
 for line in lines:
     line = unicodedata.normalize('NFC', line.strip())
+    if args.rtl:
+        line = line.translate(str.maketrans("()[]{}»«><", ")(][}{«»<>"))
+        line = bidi.algorithm.get_display(line)
```

Comment:
I made a short test and think that `bidi.algorithm.get_display(line)` could be used unconditionally below. That would simplify the handling a lot.

Reply:
Yes, that would simplify.